src/lib/components/composites/paper-components/PaperEntry.svelte

```diff
@@ -0,0 +1,91 @@
+<script lang="ts">
+    import PaperInfo from "$lib/components/composites/paper-components/PaperInfo.svelte";
+    import { type Paper, ReviewDecision, ReviewProgress, type ReviewStatus } from "$lib/types";
+    import UserAvatar from "$lib/components/composites/user-avatar/UserAvatar.svelte";
+    import { goto } from "$app/navigation";
+
+    interface PaperEntryProps {
+        paper: Paper;
+        projectId: number;
+        showReviewStatus?: boolean;
+        onClick?: () => void;
+    }
+
+    const navigateToPaperView = () => goto(`/project/${projectId}/paper/${paper.id}`);
+
+    const {
+        paper,
+        projectId,
+        showReviewStatus = false,
+        onClick = navigateToPaperView,
+    }: PaperEntryProps = $props();
+
+    // Mapping of review status to border color of paper entry
+    const reviewStatusColor: Record<ReviewStatus, string> = {
+        [ReviewProgress.Unreviewed]: "border-unreviewed-gray",
+        [ReviewDecision.Accepted]: "border-accept-green",
+        [ReviewDecision.Maybe]: "border-maybe-yellow",
+        [ReviewDecision.Declined]: "border-decline-red",
+    };
+
+    const showStatus = () => showReviewStatus && paper.reviewData !== undefined;
```

Comment:
Why does this have to be a method and not just a boolean value?
```suggestion
    const isStatusVisible = showReviewStatus && paper.reviewData !== undefined;
```

Reply:
It doesn't have to, I changed it.